### PR TITLE
EDU-2640: Moves "Install the Temporal CLI" to the top of the page

### PR DESCRIPTION
**Pull request by fairlydurable**
@tlalfano request

### CODE REVIEW

**Comment by fairlydurable** [comment]
@tlalfano Please review by end of business 14 June to better allow our team to move forward with our internal reviews. Thanks!

**Review by tlalfano — APPROVED** [review]
Looks great, new position is perfect.

File: docs/cli/index.mdx [hunk 1]
```diff
@@ -33,7 +33,70 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
...
-### Command set
+## Install the Temporal CLI {#install}
+
+**How to download and install the Temporal CLI**
+
+The Temporal CLI is available on MacOS, Windows, and Linux.
+
+### MacOS
+
+**How to install the Temporal CLI on Mac OS**
```
Comment (napcs) [inline docs/cli/index.mdx:44]:
```suggestion
```

File: docs/cli/index.mdx [hunk 2]
```diff
@@ -33,7 +33,70 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
...
 :::
 
-### Command set
+## Install the Temporal CLI {#install}
+
+**How to download and install the Temporal CLI**
+
+The Temporal CLI is available on MacOS, Windows, and Linux.
+
+### MacOS
```
Comment (napcs) [inline docs/cli/index.mdx:42]: Let's take the time to fix this now.

```suggestion
### How to install the Temporal CLI on Mac OS
```

File: docs/cli/index.mdx [hunk 3]
```diff
@@ -33,7 +33,70 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
...
+  - Download for Darwin amd64: https://temporal.download/cli/archive/latest?platform=darwin&arch=amd64
+  - Download for Darwin arm64: https://temporal.download/cli/archive/latest?platform=darwin&arch=arm64
+
+  2. Extract the downloaded archive.
+
+  3. Add the Temporal CLI binary to your PATH.
+
+### Linux
+
+**How to install the Temporal CLI on Linux**
```
Comment (napcs) [inline docs/cli/index.mdx:67]:
```suggestion
```

File: docs/cli/index.mdx [hunk 4]
```diff
@@ -33,7 +33,70 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
...
+  1. Select the platform and architecture needed.
+
+  - Download for Darwin amd64: https://temporal.download/cli/archive/latest?platform=darwin&arch=amd64
+  - Download for Darwin arm64: https://temporal.download/cli/archive/latest?platform=darwin&arch=arm64
+
+  2. Extract the downloaded archive.
+
+  3. Add the Temporal CLI binary to your PATH.
+
+### Linux
```
Comment (napcs) [inline docs/cli/index.mdx:65]:
```suggestion
### How to install the Temporal CLI on Linux
```

File: docs/cli/index.mdx [hunk 5]
```diff
@@ -33,7 +33,70 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
...
+  - Download for Linux amd64: https://temporal.download/cli/archive/latest?platform=linux&arch=amd64
+  - Download for Linux arm64: https://temporal.download/cli/archive/latest?platform=linux&arch=arm64
+
+  2. Extract the downloaded archive.
+
+  3. Add the `temporal` binary to your PATH.
+
+### Windows
+
+**How to install the Temporal CLI on Windows**
```
Comment (napcs) [inline docs/cli/index.mdx:84]:
```suggestion
```

File: docs/cli/index.mdx [hunk 6]
```diff
@@ -33,7 +33,70 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
...
+  1. Select the platform and architecture needed.
+
+  - Download for Linux amd64: https://temporal.download/cli/archive/latest?platform=linux&arch=amd64
+  - Download for Linux arm64: https://temporal.download/cli/archive/latest?platform=linux&arch=arm64
+
+  2. Extract the downloaded archive.
+
+  3. Add the `temporal` binary to your PATH.
+
+### Windows
```
Comment (napcs) [inline docs/cli/index.mdx:82]:
```suggestion
### How to install the Temporal CLI on Windows
```

File: docs/cli/index.mdx [hunk 7]
```diff
@@ -44,6 +107,9 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
 - [temporal task-queue](/cli/task-queue/)
 - [temporal workflow](/cli/workflow/)
 
+
+## Configuration
```
Comment (napcs) [inline docs/cli/index.mdx:111]: I understand why we'd add this heading here, but without something between the h2 and h3 heading it looks off. Is there something we can add here as an intro? If not, feel free to ignore.

**Review by napcs — CHANGES_REQUESTED** [review]

File: docs/cli/index.mdx [hunk 8]
```diff
@@ -44,6 +104,9 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
...
```
Comment (fairlydurable) [inline docs/cli/index.mdx:108]:
```suggestion
## Configuration

The following information provides important configuration details.
```

File: docs/cli/index.mdx (hunk 7 above)
Comment (fairlydurable) [reply docs/cli/index.mdx:108]: I agree. I added a very lame sentence but it's something.

File: docs/cli/index.mdx [hunk 9]
```diff
@@ -33,7 +33,67 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
...
+
+  - Download for Darwin amd64: https://temporal.download/cli/archive/latest?platform=darwin&arch=amd64
+  - Download for Darwin arm64: https://temporal.download/cli/archive/latest?platform=darwin&arch=arm64
+
+  2. Extract the downloaded archive.
+
+  3. Add the Temporal CLI binary to your PATH.
+
+### How to install the Temporal CLI on Linux
+
```
Comment (fairlydurable) [inline docs/cli/index.mdx:65]:
```suggestion
```

File: docs/cli/index.mdx [hunk 10]
```diff
@@ -33,7 +33,67 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
...
+  - Download for Linux amd64: https://temporal.download/cli/archive/latest?platform=linux&arch=amd64
+  - Download for Linux arm64: https://temporal.download/cli/archive/latest?platform=linux&arch=arm64
+
+  2. Extract the downloaded archive.
+
+  3. Add the `temporal` binary to your PATH.
+
+### How to install the Temporal CLI on Windows
+
```
Comment (fairlydurable) [inline docs/cli/index.mdx:80]:
```suggestion
```

File: docs/cli/index.mdx [hunk 11]
```diff
@@ -33,7 +33,67 @@ For details, see [CLI release notes](https://github.com/temporalio/cli/releases/
...
 
-### Command set
+## Install the Temporal CLI {#install}
+
+**How to download and install the Temporal CLI**
+
+The Temporal CLI is available on MacOS, Windows, and Linux.
+
+### How to install the Temporal CLI on Mac OS
+
```
Comment (fairlydurable) [inline docs/cli/index.mdx:43]:
```suggestion
```

**Review by napcs — APPROVED** [review]

**Pull request merged**